11ty/definitions/barbarian.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Barbarian
```

Comment:
This is pedantic, but 'barbarian' is a noun, and all the alt-words are adjectives. 

One solution is to make it barbaric instead.